drivers/sensor/apds9960/Kconfig

```diff
@@ -0,0 +1,41 @@
+# Kconfig - APDS9960 sensor configuration options
+
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig APDS9960
+        bool
+        prompt "APDS9960 Sensor"
+        depends on SENSOR && I2C && GPIO
+        default n
+        help
+          Enable driver for APDS9960 sensors.
+
+config APDS9960_I2C_DEV_NAME
+        string
+        prompt "I2C Master"
+        depends on APDS9960
+        default "I2C_0"
```

Comment:
Just replace default "I2C_0" with default I2C_0_NAME.
 
I2C_0_NAME is a config for the I2C_0 name, so use that instead of the string directly.